src/Streams.cpp

```diff
@@ -1,7 +1,7 @@
 #include "Streams.h"
 #include "Stream.h"
 
-using namespace BrightnESS::ForwardEpicsToKafka;
+using namespace Forwarder;
```

Comment:
This is different to the other files which tend to use `namespace Forwarder {`.
LLVM says using `using namespace Forwarder` is okay for .cpp files, as it reduces indentation.
I think we should change the others to match this.

Reply:
The LLVM format doesn't use indentation for namespaces, so the reasoning seems strange. They are also not equivalent, there are some uses `extern` for example where we'd end up with both bits of syntax. As it is not as straightforward as just swapping syntax I suggest it goes in a different PR if we make this change. If LLVM doesn't dictate that you must do it that way, I'm not convinced it is worth the time.

Reply:
Fair enough, let's change this one to match the others and be done with it.